sdk/monitor/azure-monitor-query/swagger/README.md

```diff
@@ -6,7 +6,7 @@ These settings apply only when `--tag=package-log` is specified on the command l
 
 ``` yaml $(tag) == 'package-log'
 use: '@autorest/java@4.0.22'
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/operationalinsights/data-plane/Microsoft.OperationalInsights/preview/2021-05-19_Preview/OperationalInsights.json
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/dba6ed1f03bda88ac6884c0a883246446cc72495/specification/operationalinsights/data-plane/Microsoft.OperationalInsights/preview/2021-05-19_Preview/OperationalInsights.json
```

Comment:
@scottaddie we should use the `raw.githubusercontent` endpoint for this to work because `github.com` returns html content while the raw link returns JSON content and autorest needs the json format.